### ISSUE TITLE
safe-shrinkwrap fails when run against project with no dependencies

### ISSUE THREAD

**Issue by guykisel** [ansble/safe-shrinkwrap#8]
I have a test project I use for validating my CI tools. When I run `safe-shrinkwrap` against this project, which has an empty `dependencies` field in its `package.json`, the tool fails:

```$ safe-shrinkwrap
Clearing NPM cache and Proceeding to reinstall before we shrinkwrap
    return Object.keys(depObject).reduce(function (result, key) {
                  ^

TypeError: Cannot convert undefined or null to object
    at cleanDependencies (/usr/local/nvm/versions/node/v6.2.0/lib/node_modules/safe-shrinkwrap/index.js:26:19)
    at /usr/local/nvm/versions/node/v6.2.0/lib/node_modules/safe-shrinkwrap/index.js:80:17
    at f (/usr/local/nvm/versions/node/v6.2.0/lib/node_modules/safe-shrinkwrap/node_modules/once/once.js:25:25)
    at Glob.<anonymous> (/usr/local/nvm/versions/node/v6.2.0/lib/node_modules/safe-shrinkwrap/node_modules/glob/glob.js:151:7)
    at emitOne (events.js:96:13)
    at Glob.emit (events.js:188:7)
    at Glob._finish (/usr/local/nvm/versions/node/v6.2.0/lib/node_modules/safe-shrinkwrap/node_modules/glob/glob.js:199:8)
    at done (/usr/local/nvm/versions/node/v6.2.0/lib/node_modules/safe-shrinkwrap/node_modules/glob/glob.js:184:14)
    at Glob._processSimple2 (/usr/local/nvm/versions/node/v6.2.0/lib/node_modules/safe-shrinkwrap/node_modules/glob/glob.js:690:12)
    at /usr/local/nvm/versions/node/v6.2.0/lib/node_modules/safe-shrinkwrap/node_modules/glob/glob.js:678:10
```

Thanks for developing this tool! It's pretty neat.

**Comment by designfrontier** [comment]
Just seeing this for some reason... will cut a new release this afternoon with the fix. Thanks!

**Comment by designfrontier** [comment]
this is fixed in version 1.4.3 which is published and ready to roll!

**Comment by efokschaner** [comment]
Thank you very much.
